judge/models/profile.py

```diff
@@ -276,3 +280,8 @@ class OrganizationRequest(models.Model):
     class Meta:
         verbose_name = _('organization join request')
         verbose_name_plural = _('organization join requests')
+
+
+class PEGUser(models.Model):
```

Comment:
Model changes should probably be placed into a separate django app to avoid problems with migrations. You should be able to create a blank app called `peg_migration` with just `models.py` containing this model and create the migrations. Then change all the imports. The view and other code can remain in the `judge` app.